frame/staking/src/pallet/impls.rs

```diff
@@ -1526,6 +1526,7 @@ impl<T: Config> StakingInterface for Pallet<T> {
 #[cfg(feature = "try-runtime")]
 impl<T: Config> Pallet<T> {
 	pub(crate) fn do_try_state(_: BlockNumberFor<T>) -> Result<(), &'static str> {
+		T::VoterList::iter().collect().all(|x| x.is_nominator());
```

Comment:
is this a function? :D

Reply:
I think you may have taken what was meant to be pseudo code is literal code

Reply:
@kianenigma yeah, I did not check this yet, but as it fails - I'll figure out how to implement it

Reply:
This works now.